acceptance-tests/src/testFixtures/java/tech/pegasys/teku/test/acceptance/dsl/TekuNode.java

```diff
@@ -73,6 +76,11 @@ private TekuNode(final SimpleHttpClient httpClient, final Network network, final
     super(network, TEKU_DOCKER_IMAGE, LOG);
     this.httpClient = httpClient;
     this.config = config;
+    final SpecConfiguration specConfig =
+        SpecConfiguration.builder()
+            .constants(ConstantsLoader.loadConstants(config.getNetworkName()))
+            .build();
```

Comment:
maybe we could make a builder option that takes network name rather than jumping through the constantsLoader a whole bunch...

Reply:
Good point - added a helper factory: https://github.com/ConsenSys/teku/pull/3620/commits/f67751db60f85b56628cd07bcbfbc563cf523201

Reply:
Removed the new factory for now based on offline discussions - will put up a separate PR related to this.